sdcm/nemesis.py

```diff
@@ -297,12 +298,14 @@ def get_list_of_methods_compatible_with_backend(
             self,
             disruptive: Optional[bool] = None,
             run_with_gemini: Optional[bool] = None,
-            networking: Optional[bool] = None) -> List[str]:
+            networking: Optional[bool] = None,
+            limited: Optional[bool] = None) -> List[str]:
         return self.get_list_of_methods_by_flags(
             disruptive=disruptive,
             run_with_gemini=run_with_gemini,
             networking=networking,
-            kubernetes=True if self._is_it_on_kubernetes() else None
+            kubernetes=(self._is_it_on_kubernetes() or None),
```

Comment:
Redundant parentheses

Reply:
The parentheses isn't necessary, it enhances the readability of the code.
I'm fine to remove them.